packages/bot-web-ui/src/stores/save-modal-store.js

```diff
@@ -19,22 +19,32 @@ export default class SaveModalStore {
         this.is_save_modal_open = !this.is_save_modal_open;
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    validateBotName(values) {
```

Comment:
Could create an arrow function here to avoid using this exception.

Reply:
I didn't knew arrow function can void this. Thanks 👍